### ISSUE TITLE
Unhandled Exception running NexusMods.App Linux x64

### ISSUE THREAD

**Issue by PickMeNow** [Nexus-Mods/NexusMods.App#804]
# Bug Report

## Summary
When trying to open the NexusMods.App via console I get an unhandled exception.

## Steps to reproduce
On Console:
$ ./NexusMods.App 

OS: 6.6.3-zen1-1-zen
DE: GNOME 45.2
WM: Mutter
NexusMods.App - v0.2.1

## What is the expected behaviour?
Open the Application and mod games.

## Other information

```
00:00:00.000 [ERROR] While looking for a game inside a Wine Prefix WinePrefix { ConfigurationDirectory = /home/me/.wine }: Unable to open HKEY_LOCAL_MACHINE\Software\GOG.com\Games
00:00:00.006 [ERROR] While looking for a game inside a Wine Prefix WinePrefix { ConfigurationDirectory = /home/me/.wine }: The manifest directory /home/me/.wine/drive_c/ProgramData/Epic/EpicGamesLauncher/Data/Manifests does not exist!
00:00:00.007 [ERROR] While looking for a game inside a Wine Prefix WinePrefix { ConfigurationDirectory = /home/me/.wine }: Manifest folder /home/me/.wine/drive_c/ProgramData/Origin/LocalContent does not exist!
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '/home/me/.var/app/com.usebottles.bottles/data/bottles/bottles'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at NexusMods.Paths.DirectoriesEnumerator..ctor(String directory, String pattern, EnumerationOptions options, IOSInformation os)
   at NexusMods.Paths.FileSystem.InternalEnumerateDirectories(AbsolutePath directory, String pattern, Boolean recursive)+MoveNext()
   at GameFinder.Wine.Bottles.BottlesWinePrefixManager.FindPrefixes()+MoveNext()
   at NexusMods.StandardGameLocators.AWineGameLocator`1.FindAllGames() in /_/src/NexusMods.StandardGameLocators/AWineGameLocator.cs:line 47
   at NexusMods.StandardGameLocators.AWineGameLocator`1.Find(IGame game)+MoveNext() in /_/src/NexusMods.StandardGameLocators/AWineGameLocator.cs:line 37
   at System.Linq.Enumerable.SelectManyIterator[TSource,TCollection,TResult](IEnumerable`1 source, Func`2 collectionSelector, Func`3 resultSelector)+MoveNext()
   at System.Linq.Enumerable.DistinctByIterator[TSource,TKey](IEnumerable`1 source, Func`2 keySelector, IEqualityComparer`1 comparer)+MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at NexusMods.DataModel.Games.AGame.get_Installations() in /_/src/NexusMods.DataModel/Games/AGame.cs:line 60
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Collections.ObjectModel.ObservableCollection`1..ctor(IEnumerable`1 collection)
   at NexusMods.App.UI.RightContent.FoundGamesViewModel.InitializeFromFound(IEnumerable`1 games) in /_/src/NexusMods.App.UI/RightContent/FoundGamesViewModel.cs:line 59
   at NexusMods.App.UI.RightContent.MyGames.MyGamesViewModel..ctor(IFoundGamesViewModel foundGames, IFoundGamesViewModel allGames, IEnumerable`1 games) in /_/src/NexusMods.App.UI/RightContent/MyGames/MyGamesViewModel.cs:line 18
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.ConstructorInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitConstructor(ConstructorCallSite constructorCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitDisposeCache(ServiceCallSite transientCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.DynamicServiceProviderEngine.<>c__DisplayClass2_0.<RealizeService>b__0(ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at NexusMods.App.UI.App.OnFrameworkInitializationCompleted() in /_/src/NexusMods.App.UI/App.axaml.cs:line 45
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, ShutdownMode shutdownMode)
   at NexusMods.App.Program.Main(String[] args) in /_/src/NexusMods.App/Program.cs:line 64
   at NexusMods.App.Program.<Main>(String[] args)
Aborted (core dumped)
```

**Comment by Sewer56** [comment]
Category: GameFinder

Therefore @erri120 assigned by default

**Comment by erri120** [comment]
Should be fixed in GameFinder 4.1.0.
